compiler/src/iree/compiler/InputConversion/Common/IREEImportPublic.cpp

```diff
@@ -19,7 +18,6 @@
 #include "iree/compiler/InputConversion/Common/Passes.h"
 #include "mlir/Dialect/Arith/IR/Arith.h"
 #include "mlir/Dialect/Func/IR/FuncOps.h"
-#include "mlir/IR/BuiltinDialect.h"
```

Comment:
Is builtin no longer used or do we get these indirectly now?

Reply:
Brought it back because it's needed for ModuleOp